posthog/session_recordings/session_recording_helpers.py

```diff
@@ -21,6 +23,12 @@
 
 # event.type
 
+EVENTS_RECEIVED_WITHOUT_BYTES_COUNTER = Counter(
+    "recording_events_received_without_bytes",
+    "We want all recording events to be received with snapshot bytes so we can simplify processing, tagged by resource type.",
```

Comment:
Just a reminder about `bytes`
https://github.com/PostHog/posthog/issues/18473
(Not a blocker).